code-challenges.js

```diff
@@ -18,9 +31,18 @@ var cubeAndSum1 = [2, 3, 4]
 var cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
+//must build out function that takes in array
+//the fucntion must be summed up and then cubbed
+const sumCubed = (array) => {
+  let reducer = array.reduce((a, b) => a + b, 0)
+  let cuber = reducer ** 3
+  return cuber
+}
```

Comment:
we didn't quite get the expected output from this function. I think perhaps the question wasn't terribly clear. we would want to cube all the numbers first, then add them together. one solution could look something like this:
```
const sumCubed = (array) => {
  let newValues = array.map(value => value ** 3)
  return newValues.reduce((a, b) => a + b, 0)
}
```